### DISCUSSION TITLE
如何连接内网环境中的代理服务器

### DISCUSSION THREAD

**Discussion by xiaozhikang0916** [docker-easyconnect/docker-easyconnect#144] [Q&A]
平时需要连接内网环境中的代理服务器进行mock、抓包等操作，本机连easyconnect的话可以直接设置代理。将代理设置为此容器的代理端口的话，需要如何再次连接到内网的代理服务器？

**Comment by moeshin** [comment]
clash tun mode

**Comment by czguang** [comment]
可以通过ssh LocalForward功能将内网的代理服务器的端口映射到本地端口，然后将代理设置为此本地端口即可。
